README.md

```diff
@@ -511,6 +511,8 @@ When running tests locally, do it from the project root:
 
 poetry run pytest test/
 
+poetry run pytest -n auto --dist loadscope test/  # parallel testing using n CPU workers
```

Comment:
The docs say:
> Tests are grouped by module for test functions and by class for test methods. 

Just to be clear, currently all our tests are in the form of test functions, right?

Reply:
Yes, so at the end it split it by files same as on CI